Manito/Manito/Screens/Interaction/OpenManittoViewController.swift

```diff
@@ -53,59 +47,67 @@ final class OpenManittoViewController: BaseViewController {
     }()
     private let titleLabel: UILabel = {
         let label = UILabel()
-        label.text = TextLiteral.openManittoViewController
         label.font = .font(.regular, ofSize: 34)
+        label.text = TextLiteral.openManittoViewControllerTitle
         return label
     }()
-    private var scrollNumberIndex = -1 {
+
+    // MARK: - property
+
+    private let openManittoService: DetailIngAPI = DetailIngAPI(apiService: APIService())
+
+    private var manittoRandomIndex = -1 {
         didSet {
             self.manittoCollectionView.reloadData()
         }
     }
+    private var friendsList: FriendList = FriendList(count: 0, members: [])
+    private var manitto: String = ""
+    private var manittoIndex = 0
+    private var roomId: String
     
     // MARK: - init
     
     init(roomId: String) {
         self.roomId = roomId
         super.init()
     }
-    
+
+    @available(*, unavailable)
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    deinit {
-        print("\(#file) is dead")
-    }
-    
     // MARK: - life cycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        requestWithFriends(roomId: roomId)
+        self.requestWithFriends(roomId: roomId)
```

Comment:
p2;

```suggestion
        self.requestWithFriends(roomId: self.roomId)
```

여긴 self 키워드가 빠져있군요.. 하하